crates/bevy_render/src/render_resource/bind_group.rs

```diff
@@ -296,9 +297,16 @@ pub trait AsBindGroup {
         render_device: &RenderDevice,
         images: &RenderAssets<GpuImage>,
         fallback_image: &FallbackImage,
+        buffers: &RenderAssets<GpuStorageBuffer>,
```

Comment:
I was already annoyed that this function signature was too long, but I guess it's not too bad. I'd kinda like if it was optional though. I've had a bunch of situations where I tried using AsBindGroup without an image/texture and it was annoying to have to provide it image stuff.

Reply:
I guess this could be a potential footgun, but I feel people being this deep inside bevy's rendering api and using it directly would know what they are doing enought that it matters a bit less.